Surface agentic commerce product feed validation & preview to merchants (#5630)
woocommerce/woocommerce-gateway-stripe: readme.txt

== Changelog ==

= 10.9.0 - xxxx-xx-xx =

> * Add - Agentic commerce: add a Feed preview in settings (plus a REST endpoint and WP-CLI command) that shows, without uploading, which products will be sent to AI agents and which are skipped for missing or invalid data

Reviewer: Nit: This could be shorter. One example is-
```suggestion
* Add - Add feed preview feature in settings and via WP-CLI command for Agentic commerce.
```

* Add - Add an Agentic Commerce checkout mode that redirects shoppers to your store to check out instead of completing the purchase in the AI agent
* Tweak - When enabling manual capture, clarify to agentic commerce merchants that Agentic Commerce purchases follow the capture setting in the Stripe agentic commerce dashboard
* Dev - Centralize agentic commerce feed scheduling in integration class

> * Tweak - Cap the Agentic Commerce feed preview scan so it stays responsive on large catalogs instead of timing out

Reviewer: Is this new changelog entry intentional? The feed preview feature itself is added in the same PR, so this separate entry is not quite needed.

Author: Ooops, not really. Fixed in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/5630/commits/381d790683172018673a9aa762b2017ce05e0b2a
